prov/usnic/src/usdf_ep_msg.c

```diff
@@ -163,12 +163,16 @@ int usdf_msg_fill_ep_attr(struct fi_info *hints, struct fi_info *fi,
 	return FI_SUCCESS;
 }
 
-int usdf_msg_fill_dom_attr(struct fi_info *hints, struct fi_info *fi)
+int usdf_msg_fill_dom_attr(uint32_t version, struct fi_info *hints, struct fi_info *fi, struct usd_device_attrs *dap)
 {
 	struct fi_domain_attr defaults;
 
 	defaults = msg_dflt_domain_attr;
-	defaults.name = strdup("usnic");
+
+	if (FI_VERSION_LE(version, FI_VERSION(1, 3)))
```

Comment:
Same comment for this version comparison / behavior difference.  If you want to put a lengthy comment / explanation in one place and have all the other places just refer to it, that would be fine (e.g., "See comment in foo.c explaining this version comparison and difference in behavior.").